cmd/cmd.go

```diff
@@ -107,6 +113,11 @@ func (o *options) Complete(args []string) error {
 		}
 	}
 
+	envVar, ok := os.LookupEnv("STERNCONFIG")
+	if ok {
+		o.configFilePath = envVar
+	}
```

Comment:
`Complete()` is called after `overrideFlagSetDefaultFromConfig()`, so it may not work as expected.

```
$ STERNCONFIG=$PWD/cmd/testdata/config-invalid.yaml dist/stern .
# It should fail, but we currently get no error.
```

Reply:
Fixed. e00a0e1